apps/ideas/filters.py

```diff
@@ -18,6 +19,7 @@
 ]
 
 ORDERING_CHOICES = [
+    ('?', _('Random')),
```

Comment:
Can we use another wording here? For the '?' as well as the 'Random'? We could use 'random' or 'daily' for the '?' (I just don't like the ? because it's so different from the other options). And for 'Random', I will add an issue.

Reply:
Continuing that discussion in https://github.com/liqd/a4-civic-europe/issues/345